### PR TITLE
Better prod html approach via transformer

### LINKED ISSUE #13: Use dart_to_js_script_rewriter instead of prod.html
Annoying to have to update two html files with changes

### PR DESCRIPTION
**Pull request by computmaxer**
Fix #13 

@ekweible

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
     <div id="app">
         <div class="app-loading"></div>
     </div>
-    <script src="packages/react/react.js"></script>
+    <script src="packages/react/react_prod.js"></script>
```
Comment (ekweible) [inline web/index.html]: Won't we still want react.js when deving?

Comment (computmaxer) [inline web/index.html]: hmm, shit good point.
guess I could make a transformer for that too

**Comment by computmaxer** [comment]
I've added a transformer that rewrites the `react` script tag to `react_prod.js` only in release mode.

It also adds GA tracking script only in release mode.

**Comment by computmaxer** [comment]
@ekweible please review

**Pull request merged**